### ISSUE TITLE
Content: Use relative links for web notifications

### ISSUE THREAD

**Issue by hbenali** [Meeds-io/meeds#2629]
**Is your feature request related to a problem? Please describe.**
Web notifications of content use full links to the article which causes issues when changing the platform domain name (Old issue).
**Describe the solution you'd like**
Like other notifications, the content application must use relative links instead to support domain name change. 

Thanks

**Comment by srenault-meeds** [comment]
@Julien-Dubois-eXo @azayati Have you seen this one?
It looks like the web notif for news need to be reviewed to follow technical guidelines

**Comment by Julien-Dubois-eXo** [comment]
No, I was not aware of it, but we could include it. We are planning to work on article notifications.

**Comment by srenault-meeds** [comment]
AFAIK, it has been improved. @Jihed525 do you confirm?

**Comment by Jihed525** [comment]
I confirm for web-notifs
